website_event_share/views/website_event.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="event_share_view" name="Share Button Snippet">
+        <xpath expr="//h4[@t-field='event.address_id']" position="after">
+            <div class="oe_share s_share text-center">
+                  <h3><span>Share </span>
+                    <a class="oe_share_facebook" t-attf-href="https://www.facebook.com/sharer/sharer.php?u={url}" target="_Blank"><i class="fa fa-facebook-square"/></a>
+                    <a class="oe_share_twitter" t-attf-href="https://twitter.com/intent/tweet?text={{event.website_meta_title}}&amp;url={url}" target="_Blank"><i class="fa fa-twitter"/></a>
+                    <a class="oe_share_linkedin" t-attf-href="http://www.linkedin.com/shareArticle?mini=true&amp;url={url}&amp;title={{event.website_meta_title}}" target="_Blank"><i class="fa fa-linkedin"/></a>
+                    <a class="oe_share_google" t-attf-href="https://plus.google.com/share?url={url}" target="_Blank"><i class="fa fa-google-plus-square"/></a>
+                    <a href="mailto:?body={url}&amp;subject={title}"><i class="fa fa-envelope-o"> </i></a>
```

Comment:
Any chances we can reuse [`website.s_share`](https://github.com/odoo/odoo/blob/668926ff4f75a66634991282ffabefc66aadf924/addons/website/views/snippets.xml#L267-L278) instead?